pkgs/tools/graphics/vulkan-tools-lunarg/gtest.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/external/googletest/googlemock/CMakeLists.txt b/external/googletest/googlemock/CMakeLists.txt
+index e7df8ec53d..869bfcb716 100644
+--- a/external/googletest/googlemock/CMakeLists.txt
++++ b/external/googletest/googlemock/CMakeLists.txt
+@@ -111,10 +111,10 @@ endif()
+ if (DEFINED CMAKE_VERSION AND NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.11")
+   target_include_directories(gmock SYSTEM INTERFACE
+     "$<BUILD_INTERFACE:${gmock_build_include_dirs}>"
+-    "$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/${CMAKE_INSTALL_INCLUDEDIR}>")
++    "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>")
```

Comment:
How about using `CMAKE_INSTALL_FULL_INCLUDEDIR`, as it's more semantic?

Reply:
Sounds great